src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/account_filter.py

```diff
@@ -47,13 +46,17 @@ def create_account_filter(client, account_name, resource_group_name, filter_name
 
 def update_account_filter(instance, start_timestamp=None, end_timestamp=None,
                           presentation_window_duration=None, live_backoff_duration=None,
-                          timescale=None, bitrate=None, tracks=None, force_end_timestamp=None):
+                          timescale=None, bitrate=None, first_quality=None, tracks=None, force_end_timestamp=None):
 
     if not instance:
         raise CLIError('The account filter resource was not found.')
 
-    if bitrate:
-        instance.first_quality = FirstQuality(bitrate=bitrate)
+    if first_quality is None:
+        if bitrate is not None:
+            first_quality = bitrate
```

Comment:
Same as above, or you could do:

```Python
if first_quality or bitrate:
  first_quality = FirstQuality(bitrate=first_quality or bitrate)
```

I actually think that would be better. :)

Reply:
Very cool. I'm making that change now, and the fix to the help text.